src/Test/Utilities/Portable/Traits/Traits.cs

```diff
@@ -48,6 +48,7 @@ public static class Features
             public const string CodeActionsAddParenthesesAroundConditionalExpressionInInterpolatedString = "CodeActions.AddParenthesesAroundConditionalExpressionInInterpolatedString";
             public const string CodeActionsAddRequiredParentheses = "CodeActions.AddRequiredParentheses";
             public const string CodeActionsAddShadows = "CodeActions.AddShadows";
+            public const string CodeActionsAddStaticModifier = "CodeActions.AddStaticModifier";
```

Comment:
Not sure what they are used for but I have mimicked other tests.

Reply:
They allow searching, grouping and executing related tests by the applied trait attribute in the Test explorer.